ide/languages.go/licenseinfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/languages/go/resources/go-blue.png</file>
+        <license ref="CC-BY-4.0" />
```

Comment:
As written below I find this problematic. This is missing the required attribution from CC-BY-SA-4.0.

I assume the logo was taken from the go lang repository. Wikimedia Commons holds a PD licensed copy: https://commons.wikimedia.org/wiki/File:Go_Logo_Blue.svg

If that is not acceptable, I'm willing to open my graphic programm and enter the two letters and choose the "right" color.

Reply:
Well it's a scaled down version the one in the bundle at: https://blog.golang.org/go-brand

The site says:
> Like this blog and all our gopher images, the slide themes are Creative Commons Attribution 4.0 licensed. 

The zip contains a `copyright.txt` with the content:
> All content within this zip file is Copyright 2018 The Go Authors. All rights reserved.

Reply:
@matthiasblaesing I've just re-read the license and the ASF 3rd Party License policy. It seems the best would be to create an icon of our own.

The main issue I see here, is that this image is modified (resized) work, in which case we should abide to the share-alike clause, which we can't do in ASF. Unmodified work could be included. I've tried to use the original SVG, but NetBeans does not like that one.

Unfortunately, I'm really clumsy with graphics stuff, other than resizing. The TOML file icon is about my limit. So I'd be happy if you could type that 2 letters. (A 16x16 PNG is needed with transparent background) I think the selected blue would fit both light and dark LaF-s.

Reply:
Done: https://github.com/matthiasblaesing/netbeans/commit/437391c84b3178ee5533488e1a98c56a88633224

I documented how the "logo" was created in the SVG.

I noticed a warning when starting (registration of GoLanguage collides with antlr3 grammar one). This is my fix:

https://github.com/matthiasblaesing/netbeans/commit/50955a9a8a71dfac214be650ae69b5adda147f2c

Feel free to pull changes from https://github.com/matthiasblaesing/netbeans/commits/pr-5297 and squash them into your work.

Reply:
@matthiasblaesing Thanks for the commits. I really appreciate them!